book/src/prefabs/prefabs_in_amethyst.md

```diff
@@ -1,323 +1,319 @@
 
-To instantiate a `Prefab`, we put a `Handle<Prefab<T>>` on an `Entity`. The `Entity` we put
-the `Handle` on is referred to as the main `Entity`, and the first entry in the list inside a
-`Prefab` refers to this `Entity`. All other entries in the list will spawn a new `Entity` on
-instantiation.
+## Representation
 
-NOTE: This means that we currently cannot target multiple existing entities from a single `Prefab`.
-This restriction is likely to be removed in the future.
+There are two representations of a prefab:
 
-The lifetime of a `Prefab` can roughly be divided into three distinct parts:
+* Stored representation, distributed alongside the application.
+* Loaded representation, used at runtime to instantiate entities with components.
 
-**Loading**
+The remainder of this page explains these at a conceptual level; subsequent pages contain guides on how to Amethyst applies this at a code level.
 
-This is the same as for all assets in Amethyst, the user initiates a load using `Loader`, a
-`Source` and a `Format`. The `Format` returns a `Prefab`, and the user is handed a `Handle<Prefab<T>>`,
-for some `T` that implements `PrefabData`.
+### The Basics
 
-**Sub asset loading**
+> **Note:** The prefab examples on this page include the [`PrefabData`] type names. These are written out for clarity. However, as per the RON specification, these are not strictly required.
 
-A `PrefabData` implementation could refer to other assets that need to be loaded asynchronously, and
-we don't want the user get a `Complete` notification on their `Progress` before everything has been
-loaded.
+In its stored form, a prefab is a serialized list of entities and their components that should be instantiated together. To begin, we will look at a simple prefab that attaches a simple component to a single entity. We will use the folllowing `Position` component:
 
-Because of this, once the `Format` have loaded the `Prefab` from the `Source`, and a `PrefabLoaderSystem`
-runs `process` on the `AssetStorage`, the system will invoke the `load_sub_assets` function on the
-`PrefabData` implementation. If any asset loads are triggered during this, they must adhere to the following
-rules:
-
-* the given `ProgressCounter` must be used as a parameter to the load function on `Loader`, so load tracking
-works correctly
-* the function must return `Ok(true)` (unless an `Error` occurred)
+```rust,no_run,noplaypen
+# extern crate amethyst;
+# extern crate derivative;
+# extern crate serde;
+# extern crate specs_derive;
+#
+# use amethyst::{
+#     assets::{Prefab, PrefabData, PrefabError},
+#     derive::PrefabData,
+#     ecs::{
+#         storage::DenseVecStorage,
+#         Component, Entity, WriteStorage,
+#     },
+#     prelude::*,
+# };
+# use derivative::Derivative;
+# use serde::{Deserialize, Serialize};
+# use specs_derive::Component;
+#
+#[derive(Clone, Copy, Component, Debug, Default, Deserialize, Serialize, PrefabData)]
+#[prefab(Component)]
```

Comment:
I couldn't find information about what `#[prefab(Component)] does exactly. I suggest to add a link to more explanations somewhere around.

Reply:
added a clarification 📑